### PR TITLE
no need to rewind the file here

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

**Comment by ph** [comment]
@jsvd can you test this fix on windows?

**Comment by jsvd** [comment]
on it

**Comment by jsvd** [comment]
LGTM

File: spec/filters/grok_spec.rb [hunk 1]
```diff
@@ -652,7 +652,7 @@
       pattern_dir = File.join(LogStash::Environment::LOGSTASH_HOME, "patterns")
       @file = Tempfile.new('grok', pattern_dir);
       @file.write('WORD \b[2-5]\b')
-      @file.rewind
+      @file.close
```
Comment (jordansissel) [inline spec/filters/grok_spec.rb]: If you close a Tempfile, ruby deletes it.

Comment (jordansissel) [inline spec/filters/grok_spec.rb]: Tempfile's behavior is why I wrote `Stud::Temporary` btw. We can refactor this to use that instead.

Comment (ph) [inline spec/filters/grok_spec.rb]: The test should be rewriting to use `Stud` I have created an issue for that #11

Comment (ph) [inline spec/filters/grok_spec.rb]: The whole file is crying to use `ruby-stud` ;)

**Pull request closed without merging**